### PR TITLE
pixterm: add page

### PR DESCRIPTION
**Pull request by CairnThePerson**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.3.1

### CODE REVIEW

File: pages/common/pixterm.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# pixterm
+
+> Image printing in the terminal.
+> See also: `chafa`, `catimg`.
+> More information: <https://github.com/eliukblau/pixterm>.
+
+- Print a static image to the terminal:
```
Comment (CleanMachine1) [inline pages/common/pixterm.md:7]:
```suggestion
- Print a static image:
```

File: pages/common/pixterm.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`pixterm {{path/to/file}}`
+
+- Use the image's original aspect ratio:
+
+`pixterm -s 2 {{path/to/file}}`
+
+- Specify a custom aspect ratio with [t]erminal [r]ows and [c]olumns:
```
Comment (CleanMachine1) [inline pages/common/pixterm.md:15]:
```suggestion
- Specify a custom aspect ratio using a specific number of [t]erminal [r]ows and [c]olumns:
```

I think this makes more sense to anyone reading it, although it is a bit longer.

File: pages/common/pixterm.md (hunk 1 above)
Comment (CairnThePerson) [reply pages/common/pixterm.md:7]: Similar suggestion by another reviewer on my `chafa` PR. I think it's important to mention that it's outputting to the terminal specifically, at least in the first example. Would this be a better clarification?

```suggestion
- Render a static image directly in the terminal:
```

Comment (CleanMachine1) [reply pages/common/pixterm.md:7]: 👍

**Review by CleanMachine1 — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thank you for your contribution @CairnThePerson

**Pull request merged**